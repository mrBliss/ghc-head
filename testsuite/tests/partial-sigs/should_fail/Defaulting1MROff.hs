{-# LANGUAGE PartialTypeSignatures #-}
{-# LANGUAGE NoMonomorphismRestriction #-}
module Defaulting1MROff where

alpha :: _
alpha = 3
