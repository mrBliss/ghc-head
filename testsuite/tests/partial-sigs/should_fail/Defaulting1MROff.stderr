
Defaulting1MROff.hs:6:1:
    No instance for (Num _)
      arising from the inferred type of alpha :: Num _ => _
    The type variable ‘_’ is ambiguous
    Note: there are several potential instances:
      instance Num Double -- Defined in ‘GHC.Float’
      instance Num Float -- Defined in ‘GHC.Float’
      instance Integral a => Num (GHC.Real.Ratio a)
        -- Defined in ‘GHC.Real’
      ...plus three others
